frontends/ol-components/package.json

```diff
@@ -17,12 +17,12 @@
     "@mui/material": "^5.15.15",
     "@mui/system": "^5.15.15",
     "@remixicon/react": "^4.2.0",
+    "@types/react-dom": "^18.3.0",
     "@types/tinycolor2": "^1.4.6",
     "api": "workspace:*",
     "iso-639-1": "^3.1.2",
     "lodash": "^4.17.21",
     "material-ui-popup-state": "^5.1.0",
-    "nuka-carousel": "^8.0.0",
```

Comment:
Nuka carousel was driving me nuts. It seems to just be broken in its latest version. For example, this is from their docs site:

<img width="1449" alt="wtf_nuka" src="https://github.com/mitodl/mit-open/assets/9010790/5cb25c34-3676-4a7c-b248-e73c9899c832">